# Array of grasp detections
std_msgs/Header header
perception/GraspDetection[] detections